helix-view/src/editor.rs

```diff
@@ -811,6 +811,8 @@ pub struct Editor {
     pub next_document_id: DocumentId,
     pub documents: BTreeMap<DocumentId, Document>,
 
+    pub last_event_is_char: bool,
```

Comment:
Some docs about why we need this (a link to this PR for example) would be nice

Reply:
Thanks for the comment, done